packages/babel-plugin-polyfill-corejs3/test/fixtures/usage-global/all-corejs-3.20/output.mjs

```diff
@@ -0,0 +1,64 @@
+import "core-js/modules/es.array.from.js";
+import "core-js/modules/es.string.iterator.js";
+import "core-js/modules/es.array.iterator.js";
+import "core-js/modules/es.map.js";
+import "core-js/modules/es.object.to-string.js";
+import "core-js/modules/web.dom-collections.iterator.js";
+import "core-js/modules/es.promise.js";
+import "core-js/modules/es.symbol.match.js";
+import "core-js/modules/es.string.match.js";
+import "core-js/modules/es.symbol.js";
+import "core-js/modules/es.symbol.description.js";
+import "core-js/modules/es.symbol.iterator.js";
+import "core-js/modules/es.regexp.exec.js";
+import "core-js/modules/web.queue-microtask.js";
+import "core-js/modules/es.global-this.js";
```

Comment:
Imports are still not sorted in the scope of one file and it could cause problems.